### PR TITLE
f-footer@2.0.0-beta.23-add-new-services

### PR DESCRIPTION
**Pull request by smilburn1990**

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -1021,6 +1021,23 @@
   dependencies:
     qwery "4.0.0"
 
+"@justeat/f-footer@file:packages/f-footer":
```
Comment (pikax) [inline yarn.lock:1245]: local packages

Comment (ashleynolan) [reply yarn.lock:1245]: Yep – this is a yalc link I think? Will need to be removed

**Review by ashleynolan — CHANGES_REQUESTED** [review]
Looks good – just needs the changes @pikax suggested

**Review by simonsandersje — DISMISSED** [review]

File: packages/f-footer/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.0.0-beta.23
+------------------------------
+*September 30, 2019*
```
Comment (pikax) [inline packages/f-footer/CHANGELOG.md]:
```suggestion
*October 8, 2019*
```

**Review by InsungMulumba — DISMISSED** [review]

**Review by pikax — DISMISSED** [review]

**Review by InsungMulumba — DISMISSED** [review]

File: packages/f-footer/src/components/LinkList.vue [hunk 3]
```diff
@@ -106,7 +105,7 @@ export default {
         },
 
         onResize () {
-            this.currentScreenWidth = window.innerWidth;
+            this.currentScreenWidth = sharedServices.getWindowHeight();
```
Comment (SachaZvetelman) [inline packages/f-footer/src/components/LinkList.vue]: Shouldn't this be `getWindowWidth?`

File: packages/f-header/src/tenants/it-IT.js [hunk 4]
```diff
@@ -17,7 +17,7 @@ export default {
             gtm: 'click_account_your_orders'
         },
         accountCredit: {
-            text: 'Credito dell\'account',
+            text: 'Credito dell',
```
Comment (pikax) [inline packages/f-header/src/tenants/it-IT.js:20]: is this intentional?

Comment (smilburn1990) [reply packages/f-header/src/tenants/it-IT.js:20]: Yep. Added to change log

**Review by SachaZvetelman — APPROVED** [review]

**Review by InsungMulumba — APPROVED** [review]

**Review by ashleynolan — APPROVED** [review]

File: packages/f-footer/src/components/LinkList.vue [hunk 5]
```diff
@@ -54,7 +54,7 @@
 
 <script>
 import { ChevronIcon } from '@justeat/f-vue-icons';
-import { throttle } from 'lodash-es';
+import sharedServices from '@justeat/f-services';
```
Comment (kevinrodrigues) [inline packages/f-footer/src/components/LinkList.vue:57]: Would it not be better to just pull out the methods you require here then use them directly below without having to reference the variable name?

`import { getWindowWidth, removeEvent, addEvent} from '@justeat/f-services'`

Comment (smilburn1990) [reply packages/f-footer/src/components/LinkList.vue:57]: @kevinrodrigues - Will add in the next PR. This is blocking me atm

**Pull request merged**